README.md

```diff
@@ -2,3 +2,19 @@
 
 # pyret-ide
 UI Components that power the pyret ide
+
+## React Storybook
+
+To use React Storybook:  
+
+```npm run storybook```  
+
+Then open a browser window to localhost:9001  
+
+To add to react storybooks you need to:  
+
+* Write the stories in the ```src/components/stories/directory```  
```

Comment:
you can use a single backtick instead of three. Also a good reference is the github markdown guide: https://guides.github.com/features/mastering-markdown/